### PR TITLE
Add sample for stop words removing.

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
Addresses #1996

### CODE REVIEW

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs [hunk 1]
```diff
@@ -250,11 +250,37 @@ public static class TextCatalog
...
             => new StopWordsRemovingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), columns, language);
 
+        /// <summary>
+        /// Removes stop words from incoming token streams in <paramref name="inputColumn"/>
+        /// and outputs the token streams without stopwords as <paramref name="outputColumn"/>.
+        /// </summary>
+        /// <param name="catalog">The text-related transform's catalog.</param>
+        /// <param name="inputColumn">The column containing text to remove stop words on.</param>
+        /// <param name="outputColumn">The column containing output text. Null means <paramref name="inputColumn"/> is replaced.</param>
+        /// <param name="stopwords">Array of words to remove.</param>
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: >// <param name="stopwords">Array of words to remove.</param> [](start = 9, length = 60)

did you want to reference the sample from here?

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs [hunk 2]
```diff
@@ -250,11 +250,37 @@ public static class TextCatalog
...
+            params string[] stopwords)
+            => new CustomStopWordsRemovingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), inputColumn, outputColumn, stopwords);
+
+        /// <summary>
+        /// Removes stop words from incoming token streams in input columns
+        /// and outputs the token streams without stop words as output columns.
+        /// </summary>
+        /// <param name="catalog">The text-related transform's catalog.</param>
+        /// <param name="columns">Pairs of columns to remove stop words on.</param>
+        /// <param name="stopwords">Array of words to remove.</param>
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: >// <param name="stopwords">Array of words to remove.</param> [](start = 9, length = 60)

did you want to reference the sample from here?

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs [hunk 3]
```diff
@@ -250,11 +250,37 @@ public static class TextCatalog
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="columns">Pairs of columns to remove stop words on.</param>
         /// <param name="language">Langauge of the input text columns <paramref name="columns"/>.</param>
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: >/// <param name="language">Langauge of the input text columns <paramref name="columns"/>.</param> [](start = 8, length = 97)

did you want to reference the sample from here?

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs [hunk 4]
```diff
@@ -237,7 +237,7 @@ public static class TextCatalog
         /// <param name="inputColumn">The column containing text to remove stop words on.</param>
         /// <param name="outputColumn">The column containing output text. Null means <paramref name="inputColumn"/> is replaced.</param>
         /// <param name="language">Langauge of the input text column <paramref name="inputColumn"/>.</param>
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: >/// <param name="language">Langauge of the input text column <paramref name="inputColumn"/>.</param> [](start = 8, length = 100)

did you want to reference the sample from here?

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: Am I doing this right?

---
In reply to: [250825867](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825867) [](ancestors = 250825867)

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs (hunk 3 above)
Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: Am I doing this right?

---
In reply to: [250825853](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825853) [](ancestors = 250825853)

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs (hunk 1 above)
Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: Am I doing this right?

---
In reply to: [250825805](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825805) [](ancestors = 250825805)

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs (hunk 2 above)
Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: Am I doing this right?

---
In reply to: [250825833](https://github.com/dotnet/machinelearning/pull/2236#discussion_r250825833) [](ancestors = 250825833)

File: src/Microsoft.ML.Transforms/Text/TextCatalog.cs [hunk 5]
```diff
@@ -237,7 +237,11 @@ public static class TextCatalog
...
-        public static StopWordsRemovingEstimator RemoveStopWords(this TransformsCatalog.TextTransforms catalog,
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/StopWordRemoverTransform.cs)]
+        /// ]]></format>
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: wrap it with <example>:


/// <example>
        /// <format type="text/markdown">
        /// <CDATA[
        /// [!code-csharp[StopWords]
        /// ]]>
        /// </format>
        /// </example>

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/TextCatalog.cs]: Thank you!

**Pull request merged**